### DISCUSSION TITLE
Error @quasar/apollo

### DISCUSSION THREAD

**Discussion by WCN-llc** [quasarframework/app-extension-apollo#111] [General] [answered]
**Describe the bug**

App • Running "@quasar/apollo" Quasar App Extension...

 App • ⚠️  Extension(@quasar/apollo): is not compatible with quasar v2.0.1. Required version: ^1.5.1

error Command failed with exit code 1.

**Comment by wreality** [comment, marked as answer]
Are you using the next build of app-extension-apollo ?

https://github.com/quasarframework/app-extension-apollo/tree/v2

Of particular note: `quasar ext add @quasar/apollo@next`

Check the docs there as well because some things have changed, and the extension isn't completely feature-complete for v2 yet.
There is progress tracking info here: https://github.com/quasarframework/app-extension-apollo/issues/51

**Comment by WCN-llc** [comment]
> Are you using the next build of app-extension-apollo ?
> 
> https://github.com/quasarframework/app-extension-apollo/tree/v2
> 
> Of particular note: `quasar ext add @quasar/apollo@next`
> 
> Check the docs there as well because some things have changed, and the extension isn't completely feature-complete for v2 yet.
> There is progress tracking info here: [quasarframework/app-extension-apollo#51](https://github.com/quasarframework/app-extension-apollo/issues/51)

Thanks for the help!
I applied your recommendations - no mistake!
